### PR TITLE
Preview action update

### PR DESCRIPTION
**Pull request by HeinKhantZaw**

### CODE REVIEW

File: .github/workflows/preview.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: Build and Deploy to Netlify
 on:
+  push:
```
Comment (shahednasser) [inline .github/workflows/preview.yml:3]: Please remove or comment out push as we don't need it.

**Comment by HeinKhantZaw** [comment]
@shahednasser I guess It's working now
IMG_20201002_040354

File: .github/workflows/preview.yml (hunk 1 above)
Comment (HeinKhantZaw) [reply .github/workflows/preview.yml:3]: ohh..sry about that, I was working with my phone in my bed as it's 4 a.m right now.

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @HeinKhantZaw for code and tool

**Pull request merged**